spec/views/preview.html.erb_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "rails_helper"
+
+describe "Rendering financial support information" do
+  let(:preview_course_page) { PageObjects::Page::Organisations::CoursePreview.new }
+
+  context "course has no financial incentives" do
+    it "renders the 'loan' partial" do
+      course = build(:course)
+
+      render "courses/preview/financial_support", course: course.decorate
+
+      preview_course_page.load(rendered)
+
+      expect(preview_course_page).to have_selector("[data-qa=course__loan_details]")
```

Comment:
I wonder if I need to asset _other_ partials are _not_ appearing as well? Possibly overkill ...

Reply:
If these view specs provide little value then let me know